src/components/sidebar/core/DatabaseDropdown.vue

```diff
@@ -11,14 +11,24 @@
       <a class="refresh" @click.prevent="refreshDatabases" :title="'Refresh Databases'">
         <i class="material-icons">refresh</i>
       </a>
+      <a class="refresh" @click.prevent="$modal.show('config-add-database')" :title="'Add Database'">
```

Comment:
I think this is in the right place, we'll probably tidy these buttons up after merge, but looks great so far.